src/components/theme/useThemeType.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {ThemeValueContext} from './ThemeValueContext';
+import {ThemeType} from './types';
+import {getThemeType} from './getThemeType';
+
+export function useThemeType(): ThemeType {
+    const {themeValue} = React.useContext(ThemeValueContext);
```

Comment:
Maybe better use hook instead of raw context? We have one: `useThemeValue`

Reply:
done